readme.md

```diff
@@ -35,15 +35,21 @@ tesseract.recognize("image.jpg", config)
 Also you can pass Buffer:
```

Comment:
Thank you for your contribution.
Could you undo the changes in ``package-lock.json`` (because we don't have any changes in the ``package.json``), please?

I think this is because we use different versions of npm. To avoid this, you can use ``npm ci`` instead of ``npm install``.